### PR TITLE
feat(devices): Devices capabilities

### PR DESCRIPTION
**Pull request by eoger**
See https://github.com/mozilla/fxa-auth-server/issues/2320 for context.

### CODE REVIEW

**Comment by eoger** [comment]
Tagging @philbooth for review since he also reviewed the db part :)

File: test/e2e/push_tests.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ describe('e2e/push', () => {
             lastAccessTime: 1449235471335,
             name: 'My Phone',
             type: 'mobile',
+            capabilities: 'pushbox',
```
Comment (philbooth) [inline test/e2e/push_tests.js]: Is this not supposed to be an array?

File: test/remote/db_tests.js [hunk 2]
```diff
@@ -165,7 +165,7 @@ describe('remote db', function() {
         })
         .then(sessions => {
           assert.equal(sessions.length, 1, 'sessions contains one item')
-          assert.equal(Object.keys(sessions[0]).length, 19, 'session has correct number of properties')
+          assert.equal(Object.keys(sessions[0]).length, 20, 'session has correct number of properties')
           assert.equal(typeof sessions[0].id, 'string', 'id property is not a buffer')
           assert.equal(sessions[0].uid, account.uid, 'uid property is correct')
           assert.ok(sessions[0].createdAt >= account.createdAt, 'createdAt property seems correct')
```
Comment (philbooth) [inline test/remote/db_tests.js]: Can you add an assertion for `sessions[0].deviceCapabilities`  in with this lot somewhere?

**Comment by eoger** [comment]
Thank you!

**Pull request merged**